install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -55,7 +55,7 @@ spec:
       rules:
         - alert: MCDRebootError
           expr: |
-             mcd_reboot_err > 0
+             mcd_reboot_err_total > 0
```

Comment:
How/when this alert would clear? As the metric is a counter, its value will only reset/disappear when the process exposing it (MCD?) restarts.